cmd/crio-status/main.go

```diff
@@ -16,13 +14,15 @@ const (
 	defaultSocket = "/var/run/crio/crio.sock"
 	idArg         = "id"
 	socketArg     = "socket"
+
+	deprecatednotice = "\n[Deprecation Warning: This command has been deprecated and will be removed in future versions. Please use `crio status` instead.]"
```

Comment:
`s/deprecatednotice/deprecatedNotice`

Reply:
@navisidhu could you address this comment and push this PR to the finish line?

Reply:
Replaced the word `deprecatednotice` with `deprecatedNotice`